modules/kubernetes/datadog/templates/datadog-operator.yaml.tpl

```diff
@@ -0,0 +1,42 @@
+
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: datadog
+ labels:
+   name              = "datadog"
+   xkf.xenit.io/kind = "platform"
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: datadog
+  namespace: datadog
+spec:
+  interval: 1m0s
+  url: "https://helm.datadoghq.com"
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: datadog-operator
+  namespace: datadog
+spec:
+  chart:
+    spec:
+      chart: datadog-operator
+      sourceRef:
+        kind: HelmRepository
+        name: datadog
+      version: 0.8.0
+  values:
+    appKey: ${app_key}
+    apiKey: ${api_key}
```

Comment:
This is going to be an issue. We need to have the operator read this from a secret, as we cannot store it in Git.

Reply:
A CSI Secret mount needs to be created which results in the following Secrets.
https://github.com/DataDog/helm-charts/blob/d9204cfb44e26cf8ab3911472930ec51a24d02d5/charts/datadog-operator/templates/deployment.yaml#L65-L78